metrics/tarantool.lua

```diff
@@ -13,6 +13,7 @@ local default_metrics = {
     fibers              = require('metrics.tarantool.fibers'),
     cpu                 = require('metrics.tarantool.cpu'),
     vinyl               = require('metrics.tarantool.vinyl'),
+    memtx               = require('metrics.tarantool.memtx'),
```

Comment:
Please also add a note in include/exclude section here:
https://github.com/tarantool/metrics/blob/master/doc/monitoring/api_reference.rst#metrics-functions